src/components/TrendChartLineSeries.js

```diff
@@ -0,0 +1,85 @@
+import { curveCardinal, line } from 'd3-shape'
+import PropTypes from 'prop-types'
+import startCase from 'lodash.startcase'
+import React from 'react'
+
+const TrendChartLineSeries = ({ color, series, showMarkers, x, y }) => {
+  const l = line()
+    .curve(curveCardinal.tension(0.5))
+    .x(d => x(d.date))
+    .y(d => y(d.rate))
+
+  return (
+    <g>
+      {series.map((d, i) =>
+        <g key={i} className={`series series-${d.place}-${d.crime}`}>
+          {d.segments.map((values, j) =>
+            <g key={j}>
+              <path
+                d={l(values)}
+                fill="none"
+                stroke={color(d.place)}
+                strokeWidth="2.5"
+                strokeDasharray={d.crime === 'rape-revised' && '10,10'}
```

Comment:
nit: i think we want the dashes to be smaller (and with less space in between); i would change this from `10,10` to `5,4`